llvm/test/CodeGen/MIR/skip-mir-comment-trailing-whitespace.mir

```diff
@@ -0,0 +1,28 @@
+# NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+# RUN: llc %s -mtriple=riscv64 -mattr=+v | FileCheck %s
+
+--- |
+  define void @test_vse8(ptr noundef %base, <vscale x 1 x i8> %value, i64 noundef %vl) {
+  ; CHECK-LABEL: test_vse8:
+  ; CHECK:       # %bb.0:
+  ; CHECK-NEXT:    vsetvli zero, a1, e8, mf8, ta, ma
+  ; CHECK-NEXT:    vse8.v v8, (a0)
+  ; CHECK-NEXT:    ret
+    call void @llvm.riscv.vse.nxv1i8.i64(<vscale x 1 x i8> %value, ptr %base, i64 %vl)
+    ret void
+  }
+
+  declare void @llvm.riscv.vse.nxv1i8.i64(<vscale x 1 x i8>, ptr nocapture, i64)
+
+---
+name:            test_vse8
+tracksRegLiveness: true
+body:             |
+  bb.0 (%ir-block.0):
+    liveins: $x10, $v8, $x11
+
+    %2:gprnox0 = COPY $x11
+    %1:vr = COPY $v8
+    %0:gpr = COPY $x10
+    PseudoVSE8_V_MF8 %1, %0, %2, 3 /* e8 */ :: (store unknown-size into %ir.base, align 1)
+    PseudoRET
```

Comment:
```suggestion
---
name:            test_vse8
tracksRegLiveness: true
body:             |
  bb.0 (%ir-block.0):
    liveins: $x10, $v8, $x11

    %2:gprnox0 = COPY $x11
    %1:vr = COPY $v8
    %0:gpr = COPY $x10
    PseudoVSE8_V_MF8 %1, %0, %2, 3 /* e8 */ :: (store unknown-size, align 1)
    PseudoRET
```

Reply:
I've updated it. Is it okay that it doesn't show generated code as CHECK lines? It isn't really important to what its testing, so I think it should be okay

Reply:
This must have checks. I think this is because it's missing the trailing ..., and update_mir_test_checks is buggy / overly permissive

Reply:
Goldbolt generated this test without the `...`. Adding the `...` does not add CHECK lines as desired.

Reply:
There' probably update_mir_test_checks bugs, you can just use manual checks

Reply:
Also godbolt is wrong and you need the trailing marker

Reply:
okay, ive got it sorted now.